server/views/components/iframed-video/index.njk

```diff
@@ -1,3 +1,4 @@
 <div class="iframed-video">
     <iframe src="{{ video.embedUrl }}" frameborder="0" class="iframed-video__iframe"></iframe>
 </div>
+{{ video.description }}
```

Comment:
Possibly not, it's part of the Video record, so stuck it there in case we wanted to do something with it